main.rb

```diff
@@ -0,0 +1,175 @@
+# rubocop:disable Style/CyclomaticComplexity
+require './book'
+require './person'
+require './teacher'
+require './student'
+require './rental'
+
+class Library
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def list_all_books
+    if @books.empty?
+      print 'No books in library'
+      return
+    end
+    @books.each do |book|
+      print "Title: #{book.title.capitalize}, Author: #{book.author.capitalize}\n"
+    end
+  end
+
+  def create_book
+    print 'Title: '
+    title = gets.chomp.capitalize
+    print 'Author: '
+    author = gets.chomp.capitalize
+    book = Book.new(title: title, author: author)
+    @books << book
+    puts "Book created successfully\n"
+  end
+
+  def list_all_people
+    if @people.empty?
+      puts 'Your Library is empty'
+      return
+    end
+    @people.each do |person|
+      print "[#{person.class.name}] Name: #{person.name.capitalize}, ID: #{person.id}, Age: #{person.age}\n"
+    end
+  end
+
+  def create_person
+    print 'Do you want to create a student (1) or a teacher (2)? [Input the number]: '
+
+    option = gets.chomp
+
+    case option
+    when '1'
+      print 'Age: '
+      age = gets.chomp
+      print 'Name: '
+      name = gets.chomp
+      print 'Classroom:'
+      classroom = gets.chomp
+      print 'Has parent permission? [Y/N]: '
+      permission_resp = gets.chomp
+      parent_permission = permission_resp.downcase == 'y'
+
+      student = Student.new(age: age, classroom: classroom, name: name, parent_permission: parent_permission)
+      @people.push(student)
+
+      puts "Person created successfuly\n"
```

Comment:
- [ ] I was able to create a teacher, but I couldn't create a student because of the error I got when I shared below.

![person](https://user-images.githubusercontent.com/53124548/201457434-318f6f52-4409-4d70-8834-3f1a57bd37fb.png)

Reply:
Addressed